"use server";

export default async function getNotifications() {
  try {

    const res = await 
  } catch (err) {
    console.log(err?.message || "Une erreur est survenue");
  }
}
